Services/UI/OJS.Services.Ui.Models/Contests/ContestFiltersServiceModel.cs

```diff
@@ -15,4 +15,6 @@ public class ContestFiltersServiceModel
     public int? PageNumber { get; set; }
 
     public int? ItemsPerPage { get; set; }
+
+    public string SortType { get; set; }
```

Comment:
Should be nullable or initialized with default value. Also might not be a bad idea to make this an enum like in FE

Reply:
I made it nullable and an enum.